lib/api/core/notifier.js

```diff
@@ -340,11 +340,16 @@ class NotifierController {
           result._source,
           result._id);
 
+        
+        const updatedField = Object.keys(request.input.body)
+          .filter(updatedField => !['_kuzzle_info'].includes(updatedField));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') "updatedField" hides or potentially hides a variable declared in an outer scope at line 344. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AVariableShadowing)